Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -695,14 +695,15 @@ def list_exfiltration_template_command(client: Client) -> CommandResults:
     return command_results
 
 
-def start_exfiltration_assessment_command(client: Client, template_id: str, agent_name: str,
+def start_exfiltration_assessment_command(client: Client, template_id: str, agent_name: str, agent_profile_name: str,
```

Comment:
The `agent_profile_name` arg It does not appear as mandatory in the yml file, so what happens if someone runs the command without providing this arg?

Reply:
changed to mandatory, since it is according to our api documentation